configuration/package_scripts/tedge/postinst

```diff
@@ -32,3 +32,4 @@ fi
 
 # Initialize the tedge
 tedge init
+tedge refresh-bridges
```

Comment:
Don't we have also to restart mosquitto once the bridge have been updated?

Reply:
Ah, that's true, we should indeed restart mosquitto. I'll add this to the command.